[CI] XPackRestIT ml/explain_data_frame_analytics tests failing in release builds

Looks like we have a few tests failing on our release builds. These are usually due to issues with feature toggles.

**Build scan**:
https://gradle-enterprise.elastic.co/s/uoyyjybqxg2t6/tests/:x-pack:plugin:yamlRestTest/org.elasticsearch.xpack.test.rest.XPackRestIT/test%20%7Bp0=ml%2Fexplain_data_frame_analytics%2FTest%20field_selection%20given%20job%7D#1

**Repro line**:
`./gradlew ':x-pack:plugin:yamlRestTest' --tests "org.elasticsearch.xpack.test.rest.XPackRestIT.test {p0=ml/explain_data_frame_analytics/Test field_selection given job}" -Dtests.seed=8B00D0FDE707AB62 -Dtests.security.manager=true -Dbuild.snapshot=false -Dtests.jvm.argline="-Dbuild.snapshot=false" -Dtests.locale=hu-HU -Dtests.timezone=Europe/Riga -Druntime.java=11 -Dtests.rest.blacklist=getting_started/10_monitor_cluster_health/*,xpack/15_basic/*,license/20_put_license/*,license/30_enterprise_license/* -Dlicense.key=$(pwd)/x-pack/plugin/core/snapshot.key`

**Reproduces locally?**:
Yes

**Applicable branches**:
`master` and `7.x`

**Failure excerpt**:
```
org.elasticsearch.xpack.test.rest.XPackRestIT > test {p0=ml/explain_data_frame_analytics/Test field_selection given body} FAILED
    java.lang.AssertionError: Failure at [ml/explain_data_frame_analytics:203]: expected [2xx] status code but api [ml.explain_data_frame_analytics] returned [500 Internal Server Error] [{"error":{"root_cause":[{"type":"exception","reason":"Input error: unexpected parameter 'early_stopping_enabled'. Please report this problem.\n","stack_trace":"org.elasticsearch.ElasticsearchException: Input error: unexpected parameter 'early_stopping_enabled'. Please report this problem.\n\tat org.elasticsearch.xpack.core.ml.utils.ExceptionsHelper.serverError(ExceptionsHelper.java:52)\n\tat org.elasticsearch.xpack.ml.process.AbstractNativeProcess.close(AbstractNativeProcess.java:194)\n\tat org.elasticsearch.xpack.ml.dataframe.process.AbstractNativeAnalyticsProcess.close(AbstractNativeAnalyticsProcess.java:64)\n\tat org.elasticsearch.xpack.ml.dataframe.process.NativeMemoryUsageEstimationProcess.close(NativeMemoryUsageEstimationProcess.java:18)\n\tat org.elasticsearch.xpack.ml.dataframe.process.MemoryUsageEstimationProcessManager.runJob(MemoryUsageEstimationProcessManager.java:103)\n\tat org.elasticsearch.xpack.ml.dataframe.process.MemoryUsageEstimationProcessManager.lambda$runJobAsync$0(MemoryUsageEstimationProcessManager.java:47)\n\tat org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingRunnable.run(ThreadContext.java:680)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\n\tat java.base/java.lang.Thread.run(Thread.java:834)\n"}],"type":"exception","reason":"[memory_usage_estimation_7441] Error while closing process [Input error: unexpected parameter 'early_stopping_enabled'. Please report this problem.\n], process errors: [Input error: unexpected parameter 'early_stopping_enabled'. Please report this problem.\n]","caused_by":{"type":"exception","reason":"Input error: unexpected parameter 'early_stopping_enabled'. Please report this problem.\n","stack_trace":"org.elasticsearch.ElasticsearchException: Input error: unexpected parameter 'early_stopping_enabled'. Please report this problem.\n\tat org.elasticsearch.xpack.core.ml.utils.ExceptionsHelper.serverError(ExceptionsHelper.java:52)\n\tat org.elasticsearch.xpack.ml.process.AbstractNativeProcess.close(AbstractNativeProcess.java:194)\n\tat org.elasticsearch.xpack.ml.dataframe.process.AbstractNativeAnalyticsProcess.close(AbstractNativeAnalyticsProcess.java:64)\n\tat org.elasticsearch.xpack.ml.dataframe.process.NativeMemoryUsageEstimationProcess.close(NativeMemoryUsageEstimationProcess.java:18)\n\tat org.elasticsearch.xpack.ml.dataframe.process.MemoryUsageEstimationProcessManager.runJob(MemoryUsageEstimationProcessManager.java:103)\n\tat org.elasticsearch.xpack.ml.dataframe.process.MemoryUsageEstimationProcessManager.lambda$runJobAsync$0(MemoryUsageEstimationProcessManager.java:47)\n\tat org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingRunnable.run(ThreadContext.java:680)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\n\tat java.base/java.lang.Thread.run(Thread.java:834)\n"},"stack_trace":"org.elasticsearch.ElasticsearchException: [memory_usage_estimation_7441] Error while closing process [Input error: unexpected parameter 'early_stopping_enabled'. Please report this problem.\n], process errors: [Input error: unexpected parameter 'early_stopping_enabled'. Please report this problem.\n]\n\tat org.elasticsearch.xpack.core.ml.utils.ExceptionsHelper.serverError(ExceptionsHelper.java:56)\n\tat org.elasticsearch.xpack.ml.dataframe.process.MemoryUsageEstimationProcessManager.runJob(MemoryUsageEstimationProcessManager.java:110)\n\tat org.elasticsearch.xpack.ml.dataframe.process.MemoryUsageEstimationProcessManager.lambda$runJobAsync$0(MemoryUsageEstimationProcessManager.java:47)\n\tat org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingRunnable.run(ThreadContext.java:680)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\n\tat java.base/java.lang.Thread.run(Thread.java:834)\nCaused by: org.elasticsearch.ElasticsearchException: Input error: unexpected parameter 'early_stopping_enabled'. Please report this problem.\n\tat org.elasticsearch.xpack.core.ml.utils.ExceptionsHelper.serverError(ExceptionsHelper.java:52)\n\tat org.elasticsearch.xpack.ml.process.AbstractNativeProcess.close(AbstractNativeProcess.java:194)\n\tat org.elasticsearch.xpack.ml.dataframe.process.AbstractNativeAnalyticsProcess.close(AbstractNativeAnalyticsProcess.java:64)\n\tat org.elasticsearch.xpack.ml.dataframe.process.NativeMemoryUsageEstimationProcess.close(NativeMemoryUsageEstimationProcess.java:18)\n\tat org.elasticsearch.xpack.ml.dataframe.process.MemoryUsageEstimationProcessManager.runJob(MemoryUsageEstimationProcessManager.java:103)\n\t... 5 more\n"},"status":500}]
```

--- a/x-pack/plugin/src/test/resources/rest-api-spec/test/ml/explain_data_frame_analytics.yml
+++ b/x-pack/plugin/src/test/resources/rest-api-spec/test/ml/explain_data_frame_analytics.yml
@@ -179,9 +179,6 @@
 
 ---
 "Test field_selection given body":
-  - skip:
-      version: "all"
-      reason: "Awaits fix: https://github.com/elastic/elasticsearch/issues/68337"
 
   - do:
       indices.create:

